server/src/connectionHandlers/onHover.ts

```diff
@@ -28,7 +28,8 @@ export async function onHoverHandler (params: HoverParams): Promise<Hover | null
 
   // Show documentation of a bitbake variable
   // Triggers on global declaration expressions like "VAR = 'foo'" and inside variable expansion like "FOO = ${VAR}" but skip the ones like "python VAR(){}"
-  const canShowHoverDefinitionForVariableName: boolean = (analyzer.getGlobalDeclarationSymbols(textDocument.uri).some((symbol) => symbol.name === word) && analyzer.isIdentifierOfVariableAssignment(params)) || analyzer.isBitBakeVariableExpansion(textDocument.uri, position.line, position.character) || analyzer.isPythonDatastoreVariable(textDocument.uri, position.line, position.character) || analyzer.isBashVariableName(textDocument.uri, position.line, position.character)
+  const bitbakeNode = analyzer.bitBakeNodeAtPoint(textDocument.uri, position.line, position.character)
```

Comment:
Would be nice to add a detailed commit message explaining why computing the bitbakeNode outside of isPythonDatastoreVariable improves the behavior. (Probably an embedded language document thing?). Maybe even in a comment above the related lines. Otherwise code seems good.